File: test/random/pipelines/A_UsingWaliPipeline/code/a_usingwalipipeline/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from a_usingwalipipeline.config.ConfigStore import *
from a_usingwalipipeline.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: testprojects/scala_main/pipelines/SCALA_BASIC/code/src/main/scala/com/scala/main/job1/graph/Reformat_1.scala
package com.scala.main.job1.graph

import io.prophecy.libs._
import com.scala.main.job1.config.ConfigStore._
import com.scala.main.job1.udfs.UDFs._
import com.scala.main.job1.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {
  def apply(spark: SparkSession, in: DataFrame): DataFrame = in
}
